raiden_libs/network/matrix/client.py

```diff
@@ -36,6 +54,7 @@ def __init__(
             sync_filter_limit,
             cache_level,
         )
+        self.api = GMatrixHttpApi(base_url, token)
         self.should_listen = False
         self.sync_thread = None
         self.greenlets: List[gevent.Greenlet] = list()
```

Comment:
Also, as with this change we have safe long-lived http connections, remove the header below